packages/edit-post/src/components/header/header-toolbar/style.scss

```diff
@@ -19,6 +19,21 @@
 			display: flex;
 		}
 	}
+	&.show-icon-labels {
+		button[aria-label] {
+			display: block;
+			&::after {
+				content: attr(aria-label);
+				display: block;
+				font-size: 12px;
+			}
+			&.is-primary {
+				&::after {
+					color: $black;
```

Comment:
This is causing the "Add block" button to have the wrong label colour:
<img width="94" alt="Screen Shot 2020-03-24 at 10 52 02 am" src="https://user-images.githubusercontent.com/8096000/77373932-98eae980-6dbd-11ea-8284-980afb78612f.png">
It shouldn't be necessary to set the color at all, as the pseudo-element will inherit it from its parent.

Reply:
Fixed this:

![image](https://user-images.githubusercontent.com/1477453/77596866-d78bbb80-6f05-11ea-86e4-91da6fb53083.png)